zarpaint/napari.yaml

```
@@ -0,0 +1,39 @@
+name: zarpaint
+schema_version: 0.1.0
+contributions:
+  commands:
+  - id: zarpaint.add_points_3d_with_alt_click
+    title: Create add_points_3d_with_alt_click
+    python_name: zarpaint:add_points_3d_with_alt_click
+  - id: zarpaint.set_axis_labels
+    title: Create set_axis_labels
+    python_name: zarpaint:set_axis_labels
+  - id: zarpaint.DimsSorter
+    title: Create Dims Sorter
+    python_name: zarpaint:DimsSorter
+  - id: zarpaint.create_labels
+    title: Create create_labels
+    python_name: zarpaint:create_labels
+  - id: zarpaint.watershed_split
+    title: Create watershed_split
```

```suggestion
    title: Split with watershed
```

As above, this doesn't actually need the widget I think.